src/cp/constraints/min_staffing.py

```diff
@@ -34,14 +35,15 @@ def create(self, model: CpModel, variables: dict[str, Variable]):
             6: "Sa",
             7: "So",
         }
-
         for day in self._days:
             weekday = weekday_abbreviations[day.isoweekday()]
 
             for required_level in self._min_staffing.keys():
                 eligible_employees = self._get_eligible_employees(required_level)
 
                 for shift in self._shifts:
+                    if shift._id in self.EXCLUDED_SHIFTS:
+                        continue
```

Comment:
Can you try what happens when we don't have the `EXCLUDED_SHIFTS` list? I think it should work...
 
Otherwise, I would prefer a flag on the shift object itself, like the `hidden` property of employees